DEPLOYMENT.md

```diff
@@ -2,24 +2,40 @@
 
 ## Encryption
 
-Lightning enforces encryption at rest for Credentials, for which an 
-encryption key must be provided when running in production.
+Lightning enforces encryption at rest for Credentials, for which an encryption
+key must be provided when running in production.
 
-The key is expected to be a randomized set of bytes, 32 long; and Base64
-encoded when setting the environment variable.
+The key is expected to be a randomized set of bytes, 32 long; and Base64 encoded
+when setting the environment variable.
 
-There is a mix task that can generate keys in the correct shape for use
-as an environment variable:
+There is a mix task that can generate keys in the correct shape for use as an
+environment variable:
 
 ```sh
 mix lightning.gen_encryption_key
 0bJ9w+hn4ebQrsCaWXuA9JY49fP9kbHmywGd5K7k+/s=
 ```
 
-Copy the key and set it as `PRIMARY_ENCRYPTION_KEY` in your environment.
+Copy your key _(NOT THIS ONE)_ and set it as `PRIMARY_ENCRYPTION_KEY` in your
+environment.
 
 ## Environment Variables
 
-- `PRIMARY_ENCRYPTION_KEY`  
-  Base64 encoded 32 character long string. See [Encryption](#encryption).
-
+Note that for secure deployments, it's recommended to use a combination of
+`secrets` and `configMaps` to generate secure environment variables.
+
+- `SECRET_KEY_BASE` - a secret key used as a base to generate secrets for
+  encrypting and signing data.
+- `PRIMARY_ENCRYPTION_KEY` - a base64 encoded 32 character long string.
+  See [Encryption](#encryption).
+- `ADAPTORS_PATH` - where you store your locally installed adaptors
+- `LIGHTNING_LISTEN_ADDRESS`": "0.0.0.0",
```

Comment:
Should probably add a description here. Something like:

> the address the web server should bind to, defaults to `0.0.0.0`

Assuming that default is correct..